_data/hosting_providers.json

```diff
@@ -2208,5 +2208,15 @@
     "plan": "",
     "reviewed": "2019.7.22",
     "note": ""
-  }
+  },
+  {
+    "name": "Aquatis",
+    "link": "https://aquatis.host/",
+    "category": "full",
+    "tutorial": "",
+    "announcement": "",
+    "plan": "",
+    "reviewed": "2019.8.31",
+    "note": ""
```

Comment:
Do you provide certificates for all of your hosting products? If not, we may want to add a note saying which products are supported.

Reply:
Yeah, all of the hosting products have https included.